src/agent/rustjail/Cargo.toml

```diff
@@ -31,6 +31,10 @@ tokio = { version = "1.2.0", features = ["sync", "io-util", "process", "time", "
 futures = "0.3"
 async-trait = "0.1.31"
 inotify = "0.9.2"
+libseccomp = { version = "0.1.3", optional = true }
 
 [dev-dependencies]
 serial_test = "0.5.0"
+
+[features]
+seccomp = ["libseccomp"]
```

Comment:
This has a CI impact: strictly, we'll need a new CI with this feature enabled. Is that something we can accommodate @GabyCT and @chavafg? In fact, really we'd need one new CI _per architecture_.

Reply:
Yes, we need to modify the CI.
Fortunately, all distributions in CI already have `libseccomp-dev` package and the agent enables seccomp capability by default.
Therefore, regarding the unit test, we just have to set the environment variables for libseccomp crate before building the agent.
e.g.
```
$ export LIBSECCOMP_LINK_TYPE=static
$ export LIBSECCOMP_LIB_PATH="the path of libseccomp.a"
```
However, when we test the seccomp feature with k8s, we have to add seccomp profile for k8s in [k8s-security-context.bats](https://github.com/kata-containers/tests/blob/main/integration/kubernetes/k8s-security-context.bats).
In that case, we also have to set `disable_guest_seccomp` as `false` in runtime's configuration like below.
`$ sudo sed -i '/^disable_guest_seccomp/ s/true/false/' /etc/kata-containers/configuration.toml`

I'm sorry if I'm wrong.